docs/account.md

```diff
@@ -1,5 +1,219 @@
+- `xpub` is the xpub if relevant. This information is redundant with the `id` and might be eventually dropped.
+- `derivationMode` identify the derivation scheme that is used. See [derivation documentation](./derivation.md). example of values: `segwit`, `unsplit`, `native_segwit`,...
+- `index` is the index of the account among a given `derivationMode`.
+- `freshAddress` is the "next" public address where a user should receive funds. In context of Bitcoin, the address will be "renewed" each time funds were received in order to allow some privacy. In other blockchains, it might never changes.
+- `freshAddressPath` represents the ["derivation path"](./derivation.md) where the `freshAddress` was taken from.
+- `freshAddresses` is an array of "future" fresh addresses. It is a generalisation of `freshAddress` and `freshAddressPath` (which will eventually deprecate them).
+- `name` is the name of the account that user have set. It is not saved or restored from any place but is defined by the user and locally saved.
+- `balance` represent the total amount of assets that this account holds.
+- `spendableBalance` represent the subset of `balance` that can be spent. Most of the time it will be equal to `balance` but this can varies in some blockchains.
+- `blockHeight` tracks the current blockchain block height.
+- `currency` is the associated crypto currency of the Account.
+- `unit` is the user defined preferred unit to view the account with. It's initialized to `currency.units[0]`.
+- `operationsCount` gives the total number of operations this account contains. This field exists because the `operations` array is not guaranteed to be complete.
+- `operations` is an array of operations sorted from the most recent to the oldest one. It might be partially containing only containing the N recent operations but can be paginated on.
+- `pendingOperations` is like `operations` but only for _optimistic updates_ of operations resulting of transactions that was just performed and not yet confirmed by the backend (not in a mempool, not in a block). This allows to prepend "greyed out" operation in an operations list UI.
+- `lastSyncDate` is the date of the last time a synchronisation was performed, in other words tracks how up to date the Account data is.
+- `subAccounts` is an optional field that is defined for accounts that can contain children accounts. This is for instance used for tokens and Tezos originated accounts.
+- `balanceHistory` is a cache that contains the historical datapoints of the balance in different range of times. It might not be present in Account and in that case, a fallback JS implementation will be used to calculate this from the operations array.
```

Comment:
```suggestion
- `balanceHistory` is a cache that contains the historical datapoints of the balance in different ranges of time. It might not be present in Account and in that case, a fallback JS implementation will be used to calculate this from the operations array.
```